style.mss

```diff
@@ -2,26 +2,335 @@ Map {
   background-color: @water-color;
 }
 
-/* A regular style. */
-@book-fonts:    "DejaVu Sans Book", "Arundina Regular", "Arundina Sans Regular", "Padauk Regular", "Khmer OS Metal Chrieng Regular",
-                "Mukti Narrow Regular", "Gargi Regular", "TSCu_Paranar Regular", "Tibetan Machine Uni Regular",
-                "Droid Sans Fallback Regular", "Unifont Medium", "unifont Medium";
-
-/* A bold style is available for almost all scripts. Bold text is heavier than regular text and can be used for emphasis.
-   Fallback is a regular style. */
-@bold-fonts:    "DejaVu Sans Bold", "Arundina Bold", "Arundina Sans Bold", "Padauk Bold", "TSCu_Paranar Bold",
-                "DejaVu Sans Book", "Arundina Regular", "Arundina Sans Regular", "Padauk Regular", "Khmer OS Metal Chrieng Regular",
-                "Mukti Narrow Regular", "gargi Medium", "TSCu_Paranar Regular", "Tibetan Machine Uni Regular",
-                "Droid Sans Fallback Regular", "Unifont Medium", "unifont Medium";
-
-/* An oblique or italic style is available for various (mostly latin-like) scripts. Oblique and italic style are not heavier than
-   regular style. But they are distinguishable from regular style. Fallback is a regular style. Because many (mostly asian) scripts
-   do not have an oblique or italic style by nature, be aware that for a considerable number of labels this style will make no
-   difference to the regular style. */
-@oblique-fonts: "DejaVu Sans Oblique", "Arundina Italic", "Arundina Sans Italic", "TSCu_Paranar Italic",
-                "DejaVu Sans Book", "Arundina Regular", "Arundina Sans Regular", "Padauk Regular", "Khmer OS Metal Chrieng Regular",
-                "Mukti Narrow Regular", "Gargi Regular", "TSCu_Paranar Regular", "Tibetan Machine Uni Regular",
-                "Droid Sans Fallback Regular", "Unifont Medium", "unifont Medium";
+
+/*
+About fonts:
+
+Noto is a font family that wants to cover most of Unicode with a harmonic
+design across various scripts. We use Noto for most text, with some support
+for backward-compatibility and Unifont as fallback.
+
+By order:
+
+1. Noto Sans is available for most scripts and it is used here. The list in is
+alphabetical order. Note that the “Noto Sans” base font MUST be before the
+“Noto Sans CJK *” fonts, because the CJK fonts contain non-Noto latin characters
+that should not be used. Noto Serif and Noto Mono are available for various
```

Comment:
Perhaps we can reword this because right above we say it's in alphabetical order